examples/vitess/vtctld-service.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: vtctld
+  labels:
+    name: vtctld
+spec:
+  ports:
+    - port: 15000
+      targetPort: 15000
+      nodePort: 30000
```

Comment:
if you don't specify nodePort (i.e type: NodePort but no nodePort setting in the svc) a random port should get picked for you